### PR TITLE
Showing category in WP details pane

### PR DESCRIPTION
**Pull request by mtakac2**
https://www.openproject.org/work_packages/14600

### CODE REVIEW

File: app/controllers/work_packages_controller.rb [hunk 1]
```diff
@@ -61,7 +61,7 @@ class WorkPackagesController < ApplicationController
                 :protect_from_unauthorized_export, :only => [:index, :all, :preview]
   before_filter :load_query, :only => :index
 
-  DEFAULT_WORK_PACKAGE_PROPERTIES = [:status, :assignee, :responsible,
+  DEFAULT_WORK_PACKAGE_PROPERTIES = [:status, :assignee, :responsible, :category,
```
Comment (mtakac2) [inline app/controllers/work_packages_controller.rb:64]: I am not really sure about this. Does anybody know about better way to do this?

Comment (hschink) [inline app/controllers/work_packages_controller.rb:64]: @mtakac this is how it's done for the core properties. So it's fine with me.

**Comment by myabc** [comment]
specs need to be added for `WorkPackageModel` and `WorkPackageRepresenter`.

File: lib/api/v3/work_packages/work_package_representer.rb [hunk 2]
```diff
@@ -196,6 +196,7 @@ def initialize(model, options = {}, *expand)
         property :raw_description, render_nil: true
         property :status, render_nil: true
         property :is_closed
+        property :category, getter: -> (*) { model.category.try(:name) }, render_nil: true
```
Comment (myabc) [inline lib/api/v3/work_packages/work_package_representer.rb]: @mtakac why the String? Why not the embedded `Category` object?

Comment (mtakac2) [inline lib/api/v3/work_packages/work_package_representer.rb]: @myabc :+1: You are right! I'll do that.

Comment (manwithtwowatches) [inline lib/api/v3/work_packages/work_package_representer.rb]: @myabc given that we just have strings for status and type why would we want to have an embedded resource for category? not that i have a problem with it but it seems inconsistent that way.

**Pull request merged**